src/riak_kv_entropy_manager.erl

```diff
@@ -956,11 +956,7 @@ add_index_exchanges(Index, State) ->
 -spec prune_exchanges([exchange()])
                      -> [exchange()].
 prune_exchanges(Exchanges) ->
-    L = [if A < B ->
-                 {A, B, IndexN};
-            true ->
-                 {B, A, IndexN}
-         end || {A, B, IndexN} <- Exchanges],
+    L = [Exchange || {A, B, _IndexN} = Exchange <- Exchanges, A < B],
```

Comment:
I have a hard time to see that in all cases in which prune_exchages is used, this will give the desired result. The original returns all tripples `{A, B, I}` and ensures that A =< B for all of them. Your solution just removes all `{A, B, I}` if A >= B.
Apparently you want this, because that's what the commit comment states. But how is function used by the rest of riak?